content/ri/release-notes/v.2.20.0.md

```diff
@@ -0,0 +1,41 @@
+- Upload values of [RedisJSON](https://redis.io/docs/stack/json/) from a file for new keys in Browser
+- Visualizations of [CLIENT LIST](https://redis.io/commands/client-list/) in Workbench
+- See filters previously used in Browser
+- Automatically display Redis keys added in Browser
+- Configure database connection timeout
+- Test database connection
+- Edit port for existing database connections
+
+### Details
+**Features and improvements**
+- [#1629](https://github.com/RedisInsight/RedisInsight/pull/1629), [#1739](https://github.com/RedisInsight/RedisInsight/pull/1739), [#1740](https://github.com/RedisInsight/RedisInsight/pull/1740), [#1781](https://github.com/RedisInsight/RedisInsight/pull/1781) Investigate and optimize your [search](https://redis.io/docs/stack/search/) and [graph](https://redis.io/docs/stack/graph/) queries with new visualizations of execution plans in Workbench. Visualizations are supported for [FT.EXPLAIN](https://redis.io/commands/ft.explain/),[FT.PROFILE](https://redis.io/commands/ft.profile/), [GRAPH.EXPLAIN](https://redis.io/commands/graph.explain/), [GRAPH.PROFILE](https://redis.io/commands/graph.profile/).
+- [#1698](https://github.com/RedisInsight/RedisInsight/pull/1698) Explore RedisInsight to discover its tools and capabilities to satisfy your use cases with the new walkthrough available for new users.
+- [#1631](https://github.com/RedisInsight/RedisInsight/pull/1631), [#1632](https://github.com/RedisInsight/RedisInsight/pull/1632) Migrate your database connections to another RedisInsight instance with the new feature to bulk export database connections to a file.
+- [#1741](https://github.com/RedisInsight/RedisInsight/pull/1741) Upload values of [RedisJSON](https://redis.io/docs/stack/json/) from a file for new keys in Browser.
+- [#1653](https://github.com/RedisInsight/RedisInsight/pull/1653) Investigate the list of client connections with a new Workbench visualizations for [CLIENT LIST](https://redis.io/commands/client-list/).
+- [#1625](https://github.com/RedisInsight/RedisInsight/pull/1625) Quickly set filters previously used in Browser by selecting them from the list of recently used filters.
+- [#1713](https://github.com/RedisInsight/RedisInsight/pull/1713) See your new Redis keys added in Browser without a need to refresh the list of keys.
+- [#1681](https://github.com/RedisInsight/RedisInsight/pull/1681), [#1692](https://github.com/RedisInsight/RedisInsight/pull/1692), [#1693](https://github.com/RedisInsight/RedisInsight/pull/1693) Avoid the timeout connection errors by configuring the connection timeout for databases added manually via host and port.
+- [#1696](https://github.com/RedisInsight/RedisInsight/pull/1696), [#1703](https://github.com/RedisInsight/RedisInsight/pull/1703) Test the database connection before adding the database.
+- [#1689](https://github.com/RedisInsight/RedisInsight/pull/1689) Change port for an existing database connection instead of adding a new one.
```

Comment:
```suggestion
- [#1689](https://github.com/RedisInsight/RedisInsight/pull/1689) Update the port of an existing database connection instead of adding a new one.
```